### PR TITLE
Extract Table Contents test case

### PR DESCRIPTION
**Pull request by Louis-Boulet-HC**

### CODE REVIEW

**Comment by Louis-Boulet-HC** [comment]
- Test case for content extraction completed
- Purity, Precision, and Recall assertions currently do not pass since there are issues with the OCR output that will be fixed (along with other bugs/changes) in a pull request later today

File: tests/resources/MknamLBhTbMkiPABhMhN5V/MknamLBhTbMkiPABhMhN5V_boxes.jpg
Comment (pgaviganHC) [inline tests/resources/MknamLBhTbMkiPABhMhN5V/MknamLBhTbMkiPABhMhN5V_boxes.jpg:1]: Is this file needed in the repository?

Comment (Louis-Boulet-HC) [reply tests/resources/MknamLBhTbMkiPABhMhN5V/MknamLBhTbMkiPABhMhN5V_boxes.jpg:1]: oh i guess not, neither is one of the excel files in there but it's more info about the test case table subject

File: tests/resources/MknamLBhTbMkiPABhMhN5V/MknamLBhTbMkiPABhMhN5V.pdf
Comment (pgaviganHC) [inline tests/resources/MknamLBhTbMkiPABhMhN5V/MknamLBhTbMkiPABhMhN5V.pdf:1]: Did moving these files around break any other tests? If so, please check those. I'm seeing a faileld test for gets_MiltiPage. I'm not sure if it is related to the migration of these files.

File: table_processing/TableMetrics.ipynb
Comment (pgaviganHC) [inline table_processing/TableMetrics.ipynb:1]: The changes in this file seem to be run based. Out of curiosity, are the functions in this notebook duplicates of code elsewhere in the repos?

**Comment by pgaviganHC** [comment]
> * Test case for content extraction completed
> * Purity, Precision, and Recall assertions currently do not pass since there are issues with the OCR output that will be fixed (along with other bugs/changes) in a pull request later today

Looks good in general. I wonder if it might make sense to review this after these other changes have been made?

File: table_processing/TableMetrics.ipynb
Comment (Louis-Boulet-HC) [reply table_processing/TableMetrics.ipynb:1]: yeah i don't recall making any changes in this notebook

and yes, table_metrics.py has the same functions which is where everything is being called from in the other scripts while the notebook exists more as a documentation for all the functions (metric name, formula, explanation, scale, code)

Comment (pgaviganHC) [reply table_processing/TableMetrics.ipynb:1]: Notebooks are funky that way. Over time, depending on where we go for documenting the code we may consider removing this (or modifying the notebook to call the functions implemented elsewhere). This change is not necessary for this PR.

File: tests/table_test.py [hunk 1]
```diff
@@ -41,18 +41,25 @@ def test_get_cropped_columns():
...
+    from table_processing.table_metrics import test_tables
+
+    t_name = 'MknamLBhTbMkiPABhMhN5V'
+    file_path = './tests/resources/' + t_name + '/'  + t_name
+    trueT = pd.read_excel(file_path+'_true.xlsx')
+    detc_table = Table_Detector(file_path+'.pdf')
+    table = detc_table.get_page_data()[0]['tables'][0]['table_content']
+    readT = Table.get_as_dataframe(table)
+    metrics_df = test_tables({t_name: [trueT,readT]})
+
```
[the remark this answers is not in the document]
Comment (Louis-Boulet-HC) [reply tests/table_test.py:56]: the assertion of the overlap metric is already running that row & column configuration test while the completeness & purity assertions check specific cell content OCR

File: tests/resources/MknamLBhTbMkiPABhMhN5V/MknamLBhTbMkiPABhMhN5V.pdf
Comment (Louis-Boulet-HC) [reply tests/resources/MknamLBhTbMkiPABhMhN5V/MknamLBhTbMkiPABhMhN5V.pdf:1]: No, turns out that error is related to the two deleted PDFs.
As a result, Ziyad will be sending in a PR fixing that somewhat soon I believe.

**Comment by pgaviganHC** [comment]
> o

Ping me once that is ready and I'll take another look. Thanks!

**Review by zgaffarHC — APPROVED** [review]
